blocks/media-border/style.css

```diff
@@ -0,0 +1,90 @@
+/**
+ * The following styles get applied both on the front of your site
+ * and in the editor.
+ *
+ * Replace them with your own styles or remove the file completely.
+ */
+
+.wp-block-mutualaidnyc-media-border {
+	display: flex;
+	flex-direction: column;
+	border: 10px solid;
+	min-height: 140px;
+}
+
+.wp-block-mutualaidnyc-media-border__media {
+	height: 100%;
+	background-size: cover;
+	margin: 0;
+	flex: 0 0 140px;
+	min-height: 140px;
+}
+
+.wp-block-mutualaidnyc-media-border__content {
+	padding: 1em 1em 0;
+	word-break: break-word;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	flex-grow: 1;
+}
+
+.wp-block-mutualaidnyc-media-border__media img {
+	position: absolute;
+	width: 1px;
+	height: 1px;
+	padding: 0;
+	margin: -1px;
+	overflow: hidden;
+	clip: rect(0, 0, 0, 0);
+	border: 0;
+}
+
+.wp-block-mutualaidnyc-media-border.has-background {
+	color: var(--color-primary-text);
+	border-color: var(--color-primary-text);
+}
+
+.wp-block-mutualaidnyc-media-border.has-secondary-color {
+	color: var(--color-accent-text);
+	border-color: var(--color-accent-border);
+}
+
+.wp-block-mutualaidnyc-media-border.has-dark-color {
+	color: var(--color-dark-text);
+	border-color: var(--color-dark-border);
+}
+
+.wp-block-mutualaidnyc-media-border.has-dark-alt-color {
+	color: var(--color-dark-alt-text);
+	border-color: var(--color-dark-alt-border);
+}
+
+.wp-block-mutualaidnyc-media-border.has-accent-background-color {
```

Comment:
This one conflicts with a 2020 class: 

<img width="710" alt="Screen Shot 2020-04-23 at 6 46 44 PM" src="https://user-images.githubusercontent.com/5375500/80156851-e6ac6a80-8592-11ea-8fea-9806176f9a5c.png">